src/contract/assembled_transaction.ts

```diff
@@ -316,19 +314,19 @@ export class AssembledTransaction<T> {
    * logic.
    */
   static Errors = {
-    ExpiredState: class ExpiredStateError extends Error { },
-    RestorationFailure: class RestoreFailureError extends Error { },
-    NeedsMoreSignatures: class NeedsMoreSignaturesError extends Error { },
-    NoSignatureNeeded: class NoSignatureNeededError extends Error { },
-    NoUnsignedNonInvokerAuthEntries: class NoUnsignedNonInvokerAuthEntriesError extends Error { },
-    NoSigner: class NoSignerError extends Error { },
-    NotYetSimulated: class NotYetSimulatedError extends Error { },
-    FakeAccount: class FakeAccountError extends Error { },
-    SimulationFailed: class SimulationFailedError extends Error { },
-    InternalWalletError: class InternalWalletError extends Error { },
-    ExternalServiceError: class ExternalServiceError extends Error { },
-    InvalidClientRequest: class InvalidClientRequestError extends Error { },
-    UserRejected: class UserRejectedError extends Error { },
+    ExpiredState: class ExpiredStateError extends Error {},
+    RestorationFailure: class RestoreFailureError extends Error {},
+    NeedsMoreSignatures: class NeedsMoreSignaturesError extends Error {},
+    NoSignatureNeeded: class NoSignatureNeededError extends Error {},
+    NoUnsignedNonInvokerAuthEntries: class NoUnsignedNonInvokerAuthEntriesError extends Error {},
+    NoSigner: class NoSignerError extends Error {},
+    NotYetSimulated: class NotYetSimulatedError extends Error {},
+    FakeAccount: class FakeAccountError extends Error {},
+    SimulationFailed: class SimulationFailedError extends Error {},
+    InternalWalletError: class InternalWalletError extends Error {},
+    ExternalServiceError: class ExternalServiceError extends Error {},
+    InvalidClientRequest: class InvalidClientRequestError extends Error {},
+    UserRejected: class UserRejectedError extends Error {},
```

Comment:
Looks like a LOT of "linting" changes got added. Please remove all of these. It makes it really hard to see the change in question.